recipe/meta.yaml

```
@@ -11,16 +11,15 @@ source:
 
 build:
   skip: true  # [py<38 or python_impl == 'pypy']
-  script: {{ PYTHON }} -m pip install . -vv
-  script_env:
-    - CMAKE_GENERATOR=Ninja
+  script: {{ PYTHON }} setup.py install -vv -G Ninja --single-version-externally-managed --record record.txt
```

Since `script_env` currently seems unreliable in passing through values which are also set by conda-build itself, why not adopt a solution which will work without `script_env`:

```suggestion
  script:
    - export CMAKE_GENERATOR=Ninja  # [unix]
    - set CMAKE_GENERATOR=Ninja  # [win]
    - {{ PYTHON }} -m pip install . -vv
```

---

Agree with this, setup.py install isn't likely to work that much longer.

I really wish conda-build would stop setting this value, especially if make is not installed.

---

Agree that this is more elegant. Thanks again @awvwgk. It's not really clear how conda-build is supposed to work in these cases so I wouldn't really call it a bug I guess.